src/BugsnagUnity/Native/Fallback/NativeClient.cs

```diff
@@ -164,5 +164,9 @@ public bool ShouldAttemptDelivery()
         {
             return true;
         }
+
+        public void AddOnSession()
+        {
```

Comment:
I'd prefer to see a comment in here to show it's intentionally unimplemented rather than something you hadn't got around to:

```
// Not required
```